src/compile/decode.js

```diff
@@ -222,10 +232,12 @@ var defaultValue = function (f, def) {
     case 'int32':
     case 'sint64':
     case 'sint32':
-      return parseInt(def || 0, 10)
-
+      if (def !== undefined) {
+        return parseInt(def, 10)
+      }
+      break
     default:
-      return null
+      return undefined
```

Comment:
...you could just remove this, returning `undefined` is what happens anyway

Reply:
I was trying to be specific.  If I remove the `default` clause the linter complains, but having a default clause with nothing in it seems silly.  Meh, have removed it.